src/app/components/Account/index.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from "react";
+import { useLocation, useHistory } from "react-router-dom";
+
+import { Form, Row, Col, Menu, Dropdown, Button, message } from "antd";
+
+import accountSvc from "../../../common/services/account.svc";
+import connectors from "../../../extension/background-script/connectors";
+import LndForm from "../Lnd";
+import LndHubForm from "../LndHub";
+import LnBitsForm from "../LnBits";
+import NativeConnectionForm from "../NativeConnection";
+
+const CONNECTORS = {
+  lnd: "LND",
+  lndHub: "LndHub",
+  lnBits: "LnBits",
+  native: "Native",
+};
+
+const Account = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const accountId = location.state && location.state.accountId;
+  const [account, setAccount] = useState({});
+  const [accountType, setAccountType] = useState("lnd");
+
+  useEffect(() => {
+    async function fetchAccount() {
+      const acc = (await accountSvc.getById(accountId)) || {};
+      setAccount(acc);
+      if (acc.type) {
+        setAccountType(acc.type);
+      }
+    }
+    fetchAccount();
+  }, [accountId]);
+
+  let connectorForm = null;
+  const submitHook = (innerForm) => {
+    connectorForm = innerForm;
+  };
+
+  const handleSubmit = async () => {
```

Comment:
this method is kinda long and does a lot, maybe we should split it it? 

validate(), then save(), and save does either an update or a create?